source/_components/honeywell.markdown

```diff
@@ -16,54 +16,72 @@ redirect_from:
  - /components/climate.honeywell/
 ---
 
+<p class='note'>
+There is a great deal of confusion over this integration. Please take the time to fully understand the causes of this confusion before you raise an issue, or submit a PR.
+</p>
+
+The `honeywell` climate platform lets you control Honeywell TCC (Total Connect Comfort) climate systems from Home Assistant. It does not integrate with Honeywell TCC alarm systems.
+
+Unfortunately, this integration is incorrectly implemented as a combination of two _distinct_ climate systems, one being US-based, the other is EU-based; they are _not_ interchangeable. Before you proceed, please be clear which client library is appropriate to your system.
```

Comment:
Remove all the opinions from this PR. Things like incorrectly or telling the user twice to read the docs.

Reply:
fair enough - can leave the first warning in, though?

Reply:
```
<p class='note'>
There is some potential confusion over this integration because it is currently implemented as a combination of two _distinct_ climate systems, one being US-based, the other is EU-based.

Importantly, they are _not_ interchangeable, so you must clear which region applies to your climate system.
 </p>

The `honeywell` climate platform lets you control Honeywell TCC (Total Connect Comfort) climate systems from Home Assistant. It does not integrate with Honeywell TCC alarm systems.
```

Reply:
ok